### PR TITLE
Add debug logging

### PR DESCRIPTION
**Pull request by jonas-lj**
This combines the changes of the following commits and PR's into and merges them into main:

- https://github.com/MystenLabs/sui/commit/a51401887edf62381e2ab322888dab1493894343
- https://github.com/MystenLabs/sui/commit/5acb4f5c3c561e117c6cebcfeaea6264279c31f5
- https://github.com/MystenLabs/sui/commit/b8bc634d312b49979b801f9d452914c333f90b95
- https://github.com/MystenLabs/sui/pull/10895
- https://github.com/MystenLabs/sui/pull/10900

### CODE REVIEW

**Review by benr-ml — APPROVED** [review]

File: crates/sui-core/src/authority_aggregator.rs [hunk 1]
```diff
@@ -1402,16 +1403,23 @@ where
         let cert_ref = &certificate;
         let threshold = self.committee.quorum_threshold();
         let validity = self.committee.validity_threshold();
-        let cert_bytes = bcs::to_bytes(cert_ref);
-        debug!(
+
+        // TODO: We show the below messages for debugging purposes re. incident #267. When this is fixed, we should remove them again.
+        info!(
```
Comment (longbowlu) [inline crates/sui-core/src/authority_aggregator.rs:1408]: nit: move this to line 1415. The first `info` exists before the incident.

File: crates/sui-types/src/crypto.rs [hunk 2]
```diff
@@ -1550,16 +1551,50 @@ impl<'a> VerificationObligation<'a> {
...
+            // This error message may be very long, so we print out the error in chunks of to avoid
+            // hitting a max log line length on the system.
+            for (i, chunk) in message
+                .as_bytes()
+                .chunks(chunk_size)
+                .map(std::str::from_utf8)
+                .enumerate()
+            {
+                warn!(
+                    "Failed to batch verify aggregated auth sig: {} (chunk {}): {}",
```
Comment (longbowlu) [inline crates/sui-types/src/crypto.rs:1588]: need to have tx_digest in every print. Otherwise it's difficult for node operators to collect them.

Comment (longbowlu) [reply crates/sui-types/src/crypto.rs:1588]: We've asked people to grep tx_digest

Comment (jonas-lj) [reply crates/sui-types/src/crypto.rs:1588]: That is handled by the debug span, re https://github.com/MystenLabs/sui/pull/10895#issuecomment-1508310473.

Comment (jonas-lj) [reply crates/sui-types/src/crypto.rs:1588]: Actually, that might only be true for debug! and not warn!.

**Comment by lxfind** [comment]
Do all the new logging need to be `info!`, or could they be `debug!`?

**Pull request merged**